dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/dispatch/executor/NettyExecutorManager.java

```diff
@@ -117,6 +117,7 @@ public Boolean execute(ExecutionContext context) throws ExecuteException {
                 doExecute(host, command);
                 success = true;
                 context.setHost(host);
+                context.getTaskInstance().setHost(host.getAddress());
```

Comment:
It seems duplicated with ExecutorDispatcher.dispatch.

Reply:
Yes, I will remove the set host in ExecutorDispatcher.dispatch.